azure/azure_style.yaml

```diff
@@ -165,3 +122,31 @@ jobs:
 
           # Exit with an error if any of the tracked files changed
           git diff --summary --exit-code
+
+
+  - job: ruff
+    pool:
+      vmImage: "ubuntu-22.04"
+    timeoutInMinutes: ${{ parameters.TIMEOUT }}
+    continueOnError: ${{ parameters.IGNORE_STYLE }}
+    condition: ${{ parameters.RUFF }}
+    steps:
+      - checkout: self
+      - checkout: azure_template
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: "3.9"
+      - script: |
+          cd ${{ parameters.REPO_NAME }}
+          pip install ruff
```

Comment:
Should we set the same version of ruff here to the one defined in .pre-commit-config.yaml?